JS/Clue.js

```diff
@@ -1,13 +1,52 @@
 class Clue {
+  constructor() {
+    this.question;
+    this.answerplace = 0; 
+    this.player = 0;
+    this.guessCount = 0;
+  }
+  checkAnswer(input) {
+    if( 'hello' != input && this.player === 3) {
+      this.player = 0;
+    } 
+    if ( round.currentClues[this.answerplace].answer !== input ) {
+     game.players[this.player].score  -= round.currentClues[this.answerplace].pointValue;
+     this.what++
+     if(this.guessCount === 3 ) {
+      alert("hello world yall all wrong")
```

Comment:
Just FYI alerts are likely being deprecated in the future because they are terrible so I would avoid using these at all costs in favor of rendering a message to the DOM.